app/javascript/gobierto_data/webapp/components/commons/Queries.vue

```diff
@@ -168,10 +192,84 @@ export default {
     toggle() {
       this.showSection = !this.showSection
     },
-   /* changeTab(value) {
-      const sqlCode = value.attributes.sql
-      this.$root.$emit('changeNavTab', sqlCode)
-    }*/
+    changeTab() {
+      this.$root.$emit('changeNavTab')
+    },
+    deleteQuery(index) {
+      this.endPointDelete = `/api/v1/data/queries/${index}`
+      axios.delete(this.endPointDelete, {
+        headers: {
+          'Content-type': 'application/json',
+          'Authorization': `${this.token}`
+        }
+      })
+
+      this.urlPath = location.origin
+      this.endPoint = '/api/v1/data/queries?filter[dataset_id]='
```

Comment:
Get use to parametize the baseUrl from a different file (or at least outside of the Vue scope)